.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy

```diff
@@ -38,14 +38,14 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Direc
     steps {
         gradle {
             rootBuildScriptDir(commonJobProperties.checkoutDir)
-            tasks(':beam-runners-direct-java:shadowJar')
-            tasks(':beam-runners-direct-java:shadowTestJar')
+            tasks(':runners:direct-java:shadowJar')
```

Comment:
Unfortunately, we reverted this on master. To be consistent, would you mind to keep the old syntax?

Same for line 42 and 48.

Reply:
Sure, I didn't know that. Fixed